gulpfile.babel.js

```diff
@@ -166,7 +183,7 @@ gulp.task('watch', [
 gulp.task('develop', sequencial('watch', 'server', 'application'));
 gulp.task('build-server', sequencial('watch', 'server'));
 
-gulp.task('live', ['livereloader', 'develop']);
-gulp.task('live-server', ['livereloader', 'build-server']);
+gulp.task('live', ['hotreload', 'develop']);
+gulp.task('live-server', ['hotreload', 'build-server']);
 
```

Comment:
This changes are mainly to simplify live reload handling & to make it significantly faster.